src/scripts/OutSystems/OSUI/Utils/LayoutPrivateOnPostMessage.ts

```diff
@@ -0,0 +1,104 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+namespace OutSystems.OSUI.Utils.LayoutPrivate {
+	abstract class OnPostMessage {
+		private static _createPhonePreviewStyle(notchValue: number) {
+			if (!notchValue) {
+				return;
+			}
+
+			const style = document.createElement('style');
+
+			style.textContent = `
+				body * {
+					user-select: none !important
+				}
+	
+				body.is-phone.android.portrait {
+					--status-bar-height: ${notchValue}px;
+				} 
+				
+				body.portrait.is-phone {
+					--os-safe-area-top: ${notchValue}px;
+				} 
+				
+				body.landscape.is-phone {
+					--os-safe-area-right: ${notchValue}px;
+					--os-safe-area-left: ${notchValue}px;
+				}
+	
+				.is-phone .active-screen.screen-container::-webkit-scrollbar, html::-webkit-scrollbar,
+				.is-phone.ios .active-screen.screen-container .content::-webkit-scrollbar, html::-webkit-scrollbar {
+					display: none;
+				}
+			`;
+
+			// Adds the is phone class to apply the correct styling
+			// because platform is replacing phone with tablet when we rotate the phone device
+			document.body.classList.add('is-phone');
+			document.body.setAttribute('data-status-bar-height', `${notchValue}px`);
```

Comment:
ui... Why this attribute and not the variable?

Reply:
The values is passed in the post message in the future if we need to adjust the size we don't need to release another OSUI version.

Reply:
I missed the fact that the variable is being set by css! 🥂